requirements/dev-requirements-py36.in

```diff
@@ -1,4 +1,19 @@
+--pre # This is needed because of the constraint on black, which otherwise results in \
+    "no version found"- error. See \
+    https://pip.pypa.io/en/stable/reference/pip_install/#requirements-file-format \
+    for more details.
```

Comment:
This and the restriction in line 9 is probably not needed anymore because of the _toml_-bump of _python-semantic-release_.

Reply:
Same holds for the Python 3.7 and 3.8 dependencies.

Reply:
Resolved already!